cmd/copyEnumeratorInit.go

```diff
@@ -52,10 +52,15 @@ func (cca *CookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 		TrailingDot: cca.trailingDot,
 	}
 	jobPartOrder.CpkOptions = cca.CpkOptions
-	jobPartOrder.PreserveSMBPermissions = cca.preservePermissions
-	jobPartOrder.PreserveSMBInfo = cca.preserveSMBInfo
-	// We set preservePOSIXProperties if the customer has explicitly asked for this in transfer or if it is just a Posix-property only transfer
-	jobPartOrder.PreservePOSIXProperties = cca.preservePOSIXProperties || (cca.ForceWrite == common.EOverwriteOption.PosixProperties())
+	if cca.isNFSCopy {
```

Comment:
False by default, yet a flag behind it is true by default. I suspect this is intentional; iirc not every container can support NFS properties?

Reply:
Maybe isNFSCopy could be inferred if preserve nfs perms is set? (assuming we're separating things and not combining them)